src/odemis/gui/win/acquisition.py

```diff
@@ -695,7 +703,11 @@ def __init__(self, parent, orig_tab_data):
         # Set parameters for tiled acq
         self.overlap = 0.2
         try:
-            stage_rng = self._main_data_model.stage.getMetadata()[model.MD_POS_ACTIVE_RANGE]
+            stage_md = self._main_data_model.stage.getMetadata()
+            # Hack: Limit active range with overview map range if it's in metadata, until it's provided in the GUI
+            if model.MD_OVERVIEW_RANGE in stage_md and {'x', 'y'}.issubset(stage_md[model.MD_OVERVIEW_RANGE].keys()):
+                stage_md[model.MD_POS_ACTIVE_RANGE].update(stage_md[model.MD_OVERVIEW_RANGE])
+            stage_rng = stage_md[model.MD_POS_ACTIVE_RANGE]
```

Comment:
This should work fine, but it's a little odd to update the stage_md, as it feels like the stage metadata will be changed. (It will not, because the dictionary is duplicated when passed via Pyro). So let's write it this way:
```python
stage_md = self._main_data_model.stage.getMetadata()
stage_rng = stage_md[model.MD_POS_ACTIVE_RANGE].copy()
if model.MD_OVERVIEW_RANGE in stage_md and {'x', 'y'}.issubset(stage_md[model.MD_OVERVIEW_RANGE].keys()):
    stage_rng.update(stage_md[model.MD_OVERVIEW_RANGE])
```